python/ml4ir/tests/test_metrics.py

```diff
@@ -1,17 +1,20 @@
 from ml4ir.tests.test_base import RankingTestBase
 from ml4ir.data.ranking_dataset import RankingDataset
 from ml4ir.model.ranking_model import RankingModel
-from ml4ir.config.features import FeatureConfig, parse_config
+from ml4ir.features.feature_config import FeatureConfig, parse_config
 import os
 import numpy as np
 
 
 # Constants
 GOLD_METRICS = {
-    "old_MRR": 0.7806604,
-    "new_MRR": 0.54550856,
     "old_ACR": 1.6669034,
-    "new_ACR": 2.549716,
+    "new_ACR": 2.3998579,
```

Comment:
aiyiyi. what value is verifying that we get -22% "improvement" in MRR?

Reply:
It is less about the improvement and more about being able to compute the metrics. This is not training a model at all. It just does a forward pass and computes metrics. This in all its weirdness, could probably qualify as an unit test if we bend the rules a bit.